CMakeLists.txt

```diff
@@ -248,7 +248,7 @@ endif()
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
 
-install(TARGETS xtensor-io
+install(TARGETS xtensor-io libdeps
         EXPORT ${PROJECT_NAME}-targets)
```

Comment:
I don't think this is the way to go. libdeps should probably be installed by highfive.

Reply:
It seems that this would cause xtensor-io to install libdeps as part of its cmake, and therefore libdeps to be installed in xtensor-io's cmake.